### ISSUE TITLE
Put hyperlinks in bibliography

### ISSUE THREAD

**Issue by quantitative-technologies** [krassowski/jupyterlab-citation-manager#45]
All my bibliography entries have URL links to arXiv, doi, a direct link to a pdf version of the paper, etc... 

However, these links are not clickable. Wouldn't it be better if they were?

**Comment by krassowski** [comment]
Thanks for the suggestion. Could you please post a screenshot of the element where links are not clickable that you are referring too?

**Comment by quantitative-technologies** [comment]
Hello. Yes, here are the first couple of entries in the bibliography. Nothing is clickable in the bibliograpy, even though these are URLs:

![Screenshot_2022-03-19_02-35-04](https://user-images.githubusercontent.com/29150871/159114903-d14b8b73-b2a5-4c2f-b5da-7c3cbc714e97.png)

**Comment by krassowski** [comment]
I get it now, thanks!

It looks like citeproc-js has a built-in, undocumented, extension which is off by default which allows to achieve that. It is called [`wrap_url_and_doi`](https://github.com/Juris-M/citeproc-js/blob/aa2683f48fe23be459f4ed3be3960e2bb56203f0/src/node_text.js#L325) and the fix may be as simple as enabling it (if user chooses to do so).

I also considered re-using the JupyterLab [autolinking code](https://github.com/jupyterlab/jupyterlab/blob/d7f1704307c2e345f247e1a411bdbae50f2a5708/packages/rendermime/src/renderers.ts#L499). It would require some modofications or a strong assumption that there is no formatting HTML in the URL as returned by cireproc-js. In either case, we must rely on implementation detail of citeproc-js, so I lean towards the first solution (enabling `wrap_url_and_doi`).

Conveniently I left a comment on how to enable development extensions:

https://github.com/krassowski/jupyterlab-citation-manager/blob/354a51ae59d5a44ea1b50c595a92357e016315d1/src/index.ts#L889

I believe that user should be able to turn it off, so we would need to have an extra setting as in the case of `linkToBibliography` which was added in #42.

**Comment by quantitative-technologies** [comment]
OK, thanks. I'm afraid I'm not familiar with `citeproc-js`, so I probably will not attempt to enable `wrap_url_and_doi` myself.

**Comment by krassowski** [comment]
This feature was just released in version 0.3.0; for now it is enabled by default (and can be disabled in settings), but in the future we can revise whether most users want it on or off by default.

**Comment by krassowski** [comment]
We could also think about styling it so that the link color is not as bright blue? Please let me know what you think.

**Comment by krassowski** [comment]
- Do you see the new setting in the Advanced Settings Editor?
- Is there any error in [web developer](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_are_browser_developer_tools#how_to_open_the_devtools_in_your_browser) console tab?

**Comment by krassowski** [comment]
Also, what version of JupyterLab are you using? I would recommend the latest 3.3 (I do not have bandwidth to support older versions on this project).

**Comment by quantitative-technologies** [comment]
I tried again and it works just fine:
 
![Screenshot_2022-03-27_13-28-17](https://user-images.githubusercontent.com/29150871/160294248-b0a00eda-e02e-4b80-9194-f87c5937bdb7.png)

I don't know what i did wrong, maybe forgot to restart `jupyter-lab`? I'm using version 3.2.9 btw. 

As far as appearance, I think it looks good as is! 

What happened though, is I found that a jupyter notebook was not suitable for sharing my work (e.g. it does not support foldable code outputs, and including images did not work nicely: HTML code cannot be hidden but is needed AFAIK to resize the image, and so on...)

So I made a `jupyter-book`. This has its own system for the bibliography, using a `.bib` file. Here is what the result looks like:

![Screenshot_2022-03-27_13-29-11](https://user-images.githubusercontent.com/29150871/160294560-98713fb9-96f8-4416-9738-858867ff8c2c.png)

The thing is I missed your plugin for adding the references to the `jupyter-book` document using `jupyter-lab`. I had to copy and paste the identifier from the `.bib` file each time. But I supposed that would be for a different plugin...
